builder/evaluator.go

```diff
@@ -74,7 +74,7 @@ func init() {
 	}
 }
 
-// internal struct, used to maintain configuration of the Dockerfile's
+// Builder is an internal struct, used to maintain configuration of the Dockerfile's
```

Comment:
Actually this is golint problem too. By writing docstring we just spoiling API with unnecessary info. At least we should create issue about it.
Our purpose not just suppress all golint warnings, but improve quality of code and documentation.